client/scss/components/_image_viewer.scss

```diff
@@ -50,3 +64,36 @@
     @include visually-hidden;
   }
 }
+
+.slideup {
```

Comment:
do you think there's sense in moving these transition classes into their own e.g. `_transitions.scss` – we might (probably?) want to share them across components?

Reply:
I had the same thought, but just put them in here as I was playing with them really. I think I will move them though.